### DISCUSSION TITLE
Interest Check for 4-player mode support

### DISCUSSION THREAD

**Discussion by toyoshim** [toyoshim/iona-us#40] [Ideas]
Technically said IONA can support 4-players behind a USB Hub, but as of due to the firmware size restriction, I don't have a plan to support it now.
If there are many users who want 4-players support, I may build a custom firmware for 4P support.

Alternative idea is to sell a next-gen PCB that supports JVS I/O cascading.

Feel free to add your comments to get prioritized.

**Comment by wrencrt** [comment]
I think that either would be awesome. The addition of being able to power via a barrel jack instead of a USB connector would also be nice on a next-gen board.

**Comment by toyoshim** [comment]
Aha.
For the next gen board, it would be nice to have a daisy chain connector, or at least I'd have a footprint for it so that users can implement when they want.
Also, I would use one more chip to improve electrical compatibility so that users should not be bothered by the signal adjustment option setting.

Regarding the power connector, I'm still inclined to use a USB as there are several benefits like it's super common approach today, and multiple mother boards have a USB port that can provides a required power for IONA. E.g. exA-Arcadia, the most famous use case today, have hidden USB ports.

FYI, I'm also thinking about yet another type of IONA that has a pin header instead of USB to realize the ideal inputs with the lowest latency.
